<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>992</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>tt's Live Helper</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/icons/res/icons/icon64.ico</normaloff>:/icons/res/icons/icon64.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBoxLiveStatus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>311</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>直播状态</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutCurrentTime" stretch="1,6">
       <item>
        <widget class="QLabel" name="labelCurrentTimeInfo">
         <property name="text">
          <string>北京时间</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelCurrentTime">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>未知时间</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutLiveStatus" stretch="1,6">
       <item>
        <widget class="QLabel" name="labelLiveStatusInfo">
         <property name="text">
          <string>直播状态</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelLiveStatus">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>未直播</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutLiveTitle" stretch="1,5,1">
       <item>
        <widget class="QLabel" name="labelLiveTitleInfo">
         <property name="text">
          <string>直播标题</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditLiveTitle">
         <property name="text">
          <string>未知</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonLiveTitleUpdate">
         <property name="text">
          <string>更新</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutLiveTime" stretch="1,6">
       <item>
        <widget class="QLabel" name="labelLiveTimeInfo">
         <property name="text">
          <string>直播时长</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelLiveTime">
         <property name="text">
          <string>未知</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutOnline" stretch="1,6">
       <item>
        <widget class="QLabel" name="labelOnlineInfo">
         <property name="text">
          <string>在线人数</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelOnline">
         <property name="text">
          <string>未知</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutArea" stretch="1,6">
       <item>
        <widget class="QLabel" name="labelAreaInfo">
         <property name="text">
          <string>当前分区</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelArea">
         <property name="text">
          <string>未知</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOpenBilibiliLiveSettings">
       <property name="text">
        <string>打开B站直播间设置</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>992</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_S">
    <property name="title">
     <string>推流(&amp;S)</string>
    </property>
    <addaction name="action_OnekeyStartLive"/>
    <addaction name="action_OnekeyStopLive"/>
    <addaction name="separator"/>
    <addaction name="action_OBSStartLive"/>
    <addaction name="action_OBSStopLive"/>
    <addaction name="separator"/>
    <addaction name="action_BilibiliStartLive"/>
    <addaction name="action_BilibiliStopLive"/>
   </widget>
   <addaction name="menu_S"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_OnekeyStartLive"/>
   <addaction name="action_OnekeyStopLive"/>
   <addaction name="separator"/>
  </widget>
  <action name="action_OnekeyStartLive">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/MainWindow/Actions/res/MainWindow/Actions/play.ico</normaloff>:/MainWindow/Actions/res/MainWindow/Actions/play.ico</iconset>
   </property>
   <property name="text">
    <string>一键开播</string>
   </property>
   <property name="toolTip">
    <string>同时设置B站开播及开始推流</string>
   </property>
  </action>
  <action name="action_OnekeyStopLive">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/MainWindow/Actions/res/MainWindow/Actions/pause.ico</normaloff>:/MainWindow/Actions/res/MainWindow/Actions/pause.ico</iconset>
   </property>
   <property name="text">
    <string>一键下播</string>
   </property>
   <property name="toolTip">
    <string>同时设置B站停止直播及停止推流</string>
   </property>
  </action>
  <action name="action_OBSStartLive">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/MainWindow/Actions/res/MainWindow/Actions/video-stream.ico</normaloff>:/MainWindow/Actions/res/MainWindow/Actions/video-stream.ico</iconset>
   </property>
   <property name="text">
    <string>开始OBS推流</string>
   </property>
   <property name="toolTip">
    <string>设置OBS开始推流</string>
   </property>
  </action>
  <action name="action_OBSStopLive">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/MainWindow/Actions/res/MainWindow/Actions/stop-video-stream.ico</normaloff>:/MainWindow/Actions/res/MainWindow/Actions/stop-video-stream.ico</iconset>
   </property>
   <property name="text">
    <string>停止OBS推流</string>
   </property>
   <property name="toolTip">
    <string>设置OBS开始推流</string>
   </property>
  </action>
  <action name="action_BilibiliStartLive">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/MainWindow/Actions/res/MainWindow/Actions/streaming.ico</normaloff>:/MainWindow/Actions/res/MainWindow/Actions/streaming.ico</iconset>
   </property>
   <property name="text">
    <string>设置开播</string>
   </property>
   <property name="toolTip">
    <string>向B站服务器请求开播</string>
   </property>
  </action>
  <action name="action_BilibiliStopLive">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/MainWindow/Actions/res/MainWindow/Actions/stop-streaming.ico</normaloff>:/MainWindow/Actions/res/MainWindow/Actions/stop-streaming.ico</iconset>
   </property>
   <property name="text">
    <string>设置关播</string>
   </property>
   <property name="toolTip">
    <string>向B站服务器请求关播</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>pushButtonLiveTitleUpdate</tabstop>
  <tabstop>lineEditLiveTitle</tabstop>
  <tabstop>pushButtonOpenBilibiliLiveSettings</tabstop>
 </tabstops>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
